src/libsystemd-network/sd-lldp.c

```diff
@@ -182,8 +178,7 @@ int lldp_handle_packet(tlv_packet *tlv, uint16_t length) {
 
                 if (type == LLDP_TYPE_END) {
                         if (len != 0) {
-                                log_lldp("TLV type end is not length 0. Length:%d received . Dropping ...",
-                                         len);
+                                log_lldp("TLV type end is not length 0. Length: '%d' received. Dropping.", len);
```

Comment:
"Length: '223' received" is even less readable than "Length:223 received". Maybe replace the whole message:
"TLV type end must be length 0 (not %d). Dropping."